### ISSUE TITLE
ILS with a manually set local demand for warpers accumulate warpers over the 10,000 limit

### ISSUE THREAD

**Issue by SBeausoleil** [Fireball14/DSP-Mods#4]
If you have station A that receives a supply of warpers from local production or remotely, and on the same planet an ILS B that is set to receive warpers locally, station A will forever send warpers to B and will go over the storage limit of B. 

I have seen stations with over 84k warpers in them.

**Comment by SBeausoleil** [comment]
![image](https://user-images.githubusercontent.com/10907577/113529135-ae7a1e00-9590-11eb-88fc-0ba494c84352.png)

**Comment by Fireball14** [comment]
Uh ohh....that's a problem. Il need some time to fix it.
For now, ill explain what happens.
You have 6th slot there that demands warpers. It has same settings as your manual slot except limit is 100.
When drone delivers warpers it will dump them in first found slot, that's always manual one. Then when ILS requests things again it will request wrappers becose  6th slots is not filled. To fix it for now, go to config and set show hidden slot to true.
And use it as normal slot for warpers. Just don't set manual slot for local demand for now. Il try to fix it after my work shift.

**Comment by Fireball14** [comment]
Fixed in version 1.0.4

**Comment by CageStooge** [comment]
I have found this bug yesterday, but was trying to determine what was going on and whether it was a bug in this plugin or some conflict with another one. I must say that 350,000 warpers really freed up my green cubes for research and not for making warpers!

Yes, I did not notice until it was 350,000 warpers as it was done on an empty little planet out of the way. I may not upgrade and consider it a "feature" and not a "bug"  lol

**Comment by Fireball14** [comment]
Lol well its up to you :) Always play the game the way you like. Its your playthrough after all.

**Comment by leng-yue** [comment]
I think this bug is not fixed

**Comment by Fireball14** [comment]
Hello, first please ensure your using 1.0.4 version. If yes than please drop me your save, ill take a look. Also mod list would really help, may be there is a conflict with some mod.

**Comment by leng-yue** [comment]
Yes, I am sure  that I am using 1.0.4
![version](https://i.loli.net/2021/04/18/QmqfTO4CrPAGikv.png)

![bug](https://i.loli.net/2021/04/18/2iFWwGQzE7LOXsI.png)
## Mod List (All mods are newest)
- DistributeSpaceWarpers
- FastDrones
- AutoStellarNavigation
- MultiBuild
- DSPMultiplyStackSize
- PlanerMinor
- BetterStats
- DSPNewGamePlus
- FreeFoundations_FreeSoil_HigherBuildRange
- BuildCounter
- RailgunsRetargeting
- SuperNightLight
- SphereProgress
- DSPHierarchicalDisplay

## Save (Archive)
[Google Drive](https://drive.google.com/file/d/1iQkemGGdnKtjWfhGGI13m7SMtIBP-dpc/view?usp=sharing)

**Comment by Fireball14** [comment]
Thx a lot for this report!
I managed to fix it. I have only slight idea how could this happen, I changed to native methods setting up slot item, I hope this will fix it for the future. Also added auto-correct for this problem if it ever happens again for any reason.

PS Heads-up,  somewhere in your save there is in total 3 stations that had this problem including one that you showed me.
Sorry again for inconvenience. Have a nice game.
